Dockerfile.local

```diff
@@ -1,15 +1,12 @@
-FROM node:8
+FROM node:12
 
 # Create our application directory
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-# Install Screwdriver API
-COPY package.json /usr/src/app/
-RUN npm install --production
-
-# Copy everything else
+# Install Screwdriver Store
 COPY . /usr/src/app
+RUN npm install --production
```

Comment:
Could you explain about these changes?
I can build image with the previous `Dockerfile.local`, which I replaced only the base image with `node:12`, in my environment.